tests/providers/google/cloud/transfers/test_sheets_to_gcs.py

```diff
@@ -114,9 +114,11 @@ def test_execute(self, mock_upload_data, mock_xcom, mock_sheet_hook, mock_gcs_ho
         )
 
         calls = [mock.call(spreadsheet_id=SPREADSHEET_ID, range_=r) for r in RANGES]
-        mock_sheet_hook.return_value.get_values.has_calls(calls)
+        mock_sheet_hook.return_value.get_values.assert_has_calls(calls)
 
         calls = [mock.call(mock_gcs_hook, mock_sheet_hook, r, v) for r, v in zip(RANGES, data)]
-        mock_upload_data.has_calls(calls)
-
+        mock_upload_data.assert_called()
```

Comment:
@uranusjr I had to split the `assert_has_calls` into two tests because the mock_calls and the calls given in the list had differences, such as `GCSHook` vs `GCSHook()`. If you can suggest a better test/tests, I am happy to implemen them.
Two assert tests I wrote are:

1. Count the number of calls made by the mock and compare with the expected calls
2. Was the mock called at least once?